src/ecies/mod.rs

```diff
@@ -14,6 +14,8 @@
 
 #![deny(missing_docs)]
 
+//! Implementation of an integrated encryption scheme.
```

Comment:
What's this doing here? :P

Reply:
This PR has been open for quite some time, so new clippy warnings crept in. I have quite a lot of PRs open so I didn't feel like making yet another one fixing this in a separate PR.